src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -27,7 +31,8 @@ public class Shooter extends SubsystemBase {
   private final DoubleLogEntry log_pid_output = new DoubleLogEntry(m_log, "/shooter/pid/output");
 
   public final Trigger m_atSetpoint;
-
+  
+  PWM lEDPwm = new PWM(0);
```

Comment:
Make this final and fix naming
```suggestion
  private final PWM m_led = new PWM(0);
```